.changes/reload.md

```diff
@@ -0,0 +1,5 @@
+---
+'wry': 'patch'
```

Comment:
```suggestion
'wry': 'minor'
```

Reply:
well @FabianLars told me above that it should be `patch` because wry is still `0.x`, so what is the policy on it?

https://github.com/tauri-apps/wry/pull/1500#discussion_r1971440463

Reply:
don't listen to lucas, he's new here :P (unless it's the android change, then it's me)

Reply:
ahhh right I'm so old